app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -16,19 +16,25 @@
 
 package uk.gov.hmrc.transitmovementsauditing.controllers
 
+import akka.stream.Materializer
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 import play.api.mvc.Action
-import play.api.mvc.AnyContent
 import play.api.mvc.ControllerComponents
+import uk.gov.hmrc.transitmovementsauditing.controllers.stream.StreamingParsers
+
 import javax.inject.Inject
 import javax.inject.Singleton
 import scala.concurrent.Future
 
 @Singleton()
-class MicroserviceHelloWorldController @Inject() (cc: ControllerComponents) extends BackendController(cc) {
+class AuditController @Inject() (cc: ControllerComponents, implicit val materializer: Materializer) extends BackendController(cc) with StreamingParsers {
 
-  def hello(): Action[AnyContent] = Action.async {
-    implicit request =>
-      Future.successful(Ok("Hello world"))
+  def post(message: String): Action[Source[ByteString, _]] = Action.async(
```

Comment:
The parameter `message` should be an audit type, at least named as such. I'm not sure how to create a parameter such that in only allows certain choices though, so it _may_ need to be a string, but I think it's in scope of this ticket to at least see if we can create an `AuditType` sealed trait and see if we can route on this type.

(e.g. `/audit/DepartureDeclaration` would select a `DepartureDeclaration` type here - if that's possible)